chemprop/models/model.py

```diff
@@ -253,5 +268,29 @@ def load_from_file(cls, model_path, map_location=None, strict=True) -> MPNN:
 
         model = cls(**hparams)
         model.load_state_dict(state_dict, strict=strict)
+        
+        model.input_scalers = InputTransform(d["input_scalers"])
+        model.output_transform = OutputTransform(d["output_scaler"])
 
         return model
+
+
+class OutputTransform(object):
```

Comment:
inheriting from `object` is poor style. The inheritance is implied by the language